nodes/geometry/intersect_line_line.py

```diff
@@ -0,0 +1,40 @@
+import bpy
+from ... base_types.node import AnimationNode
+
+class IntersectLineLineNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_IntersectLineLineNode"
+    bl_label = "Intersect Line Line"
+    bl_width_default = 160
+    searchTags = ["Closest Points on 2 Lines"]
+        
+    def create(self):
```

Comment:
why do you use such uncommon default values? Do they have a special meaning?
Maybe the standart vectors (0, 0, 0) as start and (1, 0, 0) / (0, 1, 0) as end would be better.

for better readability I'd prefer the identifiers lineStart1, lineEnd1, ... what do you think?